client/src/components/Calendar.jsx

```diff
@@ -0,0 +1,12 @@
+// import React from 'react';
```

Comment:
where is the timedown module? I (nor Kevin) can find it. Other than that, your variable names look adequate.

Reply:
I found it. It's in the second commit. I didn't realize it would split them up.

Reply:
It looks like it included everything from PR #6 in one commit and then the new stuff in #7.